test-configs.yaml

```diff
@@ -581,6 +581,15 @@ device_types:
       - blacklist: *allmodconfig_filter
       - blacklist: {kernel: ['v3.']}
 
+  qcom_qdf2400:
+    name: 'qcom-qdf2400'
+    mach: qcom
+    arch: arm64
+    boot_method: grub
+    filters:
+      - whitelist: {defconfig: ['defconfig']}
```

Comment:
ooi why only boot test with defconfig on this platform?

Reply:
Sure I meant, is there any reason to not boot test other configs than the plain `defconfig`?

Reply:
Oh, no probably not but it's the only config I've tested and it's enough for now.
Any objections for me to merge?

Reply:
Alright then :)

Note: Generally speaking, I believe we should be testing all the configs and test plans that we can unless we're hitting some capacity limit.  If you're OK with that for this device then maybe you could add more test plans and widen the filters to other configs at some point.